src/build/generate-contributions.ts

```diff
@@ -290,6 +292,43 @@ const nodeAttachConfig: IDebugger<INodeAttachConfiguration> = {
   ],
   configurationAttributes: {
     ...nodeBaseConfigurationAttributes,
+    restart: {
+      description: refString('node.attach.restart.description'),
+      default: true,
+      oneOf: [
+        {
+          type: 'boolean',
+        },
+        {
+          type: 'object',
+          required: ['exponential'],
```

Comment:
All these options kind of seem like overkill, but sure :)

Reply:
Eh probably 😛  We'll get telemetry on who set them so we can always yank them if we see no one cases about it.